### PR TITLE
build: add python3-pip to ubuntu.Dockerfiles

### PR DESCRIPTION
**Pull request by cclauss**
https://pypi.org/project/tap2junit says that __tap2junit__ is compatible with legacy Python but says nothing about Python 3 compatibility.  Related to #1674

### CODE REVIEW

**Comment by thefourtheye** [comment]
cc @nodejs/build

**Comment by cclauss** [comment]
@refack @rvagg @richardlau Is this PR going in the right direction or should I close it?

**Comment by rvagg** [comment]
I have no idea @cclauss, you tell me, what does this even do? What does it mean to have both `python-pip` and `python3-pip` and to install `tap2junit` with both? What will that do when it's invoked by different versions of Node? I can merge this and rebuild the images but I'd be doing it completely blind to the consequences and I don't like having to scramble to fix something that's broken because I didn't understand it.

**Comment by cclauss** [comment]
__Python 2__ and __Python 3__ have two _separate_ copies of __pip__.  These are usually called __pip2__ and __pip3__ but not always so a safe approach is to do __python 2 -m pip__ and __python3 -m pip__ to ensure that you are getting a hold of the pip that corresponds to the Python you are using.

TL;DR They maintain _separate_ "site-packages" directories so that a module like __tap2junit__ needs to be installed into both if we want that module to be accessible to both Py2 and Py3.

On my machine I get:
```
$ python2 -m pip --version
pip 19.1.1 from /usr/local/lib/python2.7/site-packages/pip (python 2.7)
$ python3 -m pip --version
pip 19.1.1 from /usr/local/lib/python3.7/site-packages/pip (python 3.7)
```
We will need to have a similar set up on all boxes where we want to test both Pythons.

**Comment by rvagg** [comment]
should be deployed across all the containers now
